### DISCUSSION TITLE
Stuck at the booting gof blackarch linux

### DISCUSSION THREAD

**Discussion by JunaithSM** [archlinux/archinstall#3047] [General]
![IMG20241224112038](https://github.com/user-attachments/assets/606842ae-b61b-4259-b019-f097e5d52cd2)

I did `sudo pacman -Syyu`
When I reboot it not booting..
The usb booting both normal and grup2 is not working.
I am searching for solutions like for 3hrs
I can just use the power button to shutdown.. tty is not opening.
I am just stuck.
I cannot do anything from here no key is working any help.

**Comment by svartkanin** [comment]
I'd suggest raising an issue instead of discussion and provide a log file
